Change dependency and delete patch that was fixed upstream in gumyr/build123d

jdegenstein opened the pull request:
build123d does not depend on cadquery but it does depend on cadquery-ocp which is just called ocp on conda. I also deleted a patch file that is no longer needed as it was fixed upstream.

jdegenstein committed:
Delete recipe/0004-Fix-raw-regex.patch

Fixed upstream

jdegenstein committed:
0005-Fix-dependency-names-for-conda-forge.patch -> build123d does not depend on cadquery, just cadquery-ocp AKA ocp on conda

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of the recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/11981089975](https://github.com/conda-forge/conda-forge-webservices/actions/runs/11981089975). Examine the logs at this URL for more detail.</sub>

ndevenish committed:
Remove patch from recipe and increase build number

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ ``requirements: run: ocp >= 7.7.2`` should not contain a space between relational operator and the version, i.e. ``ocp >=7.7.2``

For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of the recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - For the `test.requires` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12528103293](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12528103293). Examine the logs at this URL for more detail.</sub>

ndevenish committed:
Remove duplicate OCP

ndevenish committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.45.1, and conda-forge-pinning 2024.12.27.18.01.51

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `host` section of the recipe, you should usually use `python {{ python_min }}` for the `python` entry.
   - For the `run` section of the recipe, you should usually use `python >={{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12528971394](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12528971394). Examine the logs at this URL for more detail.</sub>

ndevenish committed:
Remove cadquery from pyproject.toml

It's installed via the ocp package on conda-forge, but this
doesn't install the pip metadata required for pip-check to work

ndevenish committed:
Update python dependency for CFEP-25

ndevenish committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.45.1, and conda-forge-pinning 2024.12.27.18.01.51

ndevenish committed:
Force minimum python to 3.12

ndevenish committed:
Fix patches by regenerating patch series

ndevenish committed:
Fix reintroduced bad name

ndevenish commented:
Thank you! I missed that the cadquery part wasn't used.
